src/Symfony/Component/HttpFoundation/README.md

```diff
@@ -4,6 +4,16 @@ HttpFoundation Component
 The HttpFoundation component defines an object-oriented layer for the HTTP
 specification.
 
+Sponsor
+-------
+
+The HttpFoundation component for Symfony 5.4/6.0 is [backed][1] by [Laravel][2].
+
+Laravel is a PHP web development framework that is passionate about maximum developer
+happiness. Laravel is built using a variety of bespoke and Symfony based components.
+
+Help Symfony by [sponsoring][3] its development!
```

Comment:
Let's use inline reference, like we use in all other links in this doc? :)

Reply:
I copy/pasted from a similar mention in the Routing readme for 5.3.